### ISSUE TITLE
onShowInboxMessage() and onShowInbox() never called for iOS

### ISSUE THREAD

**Issue by asegurola** [urbanairship/airship-flutter#106]
# Preliminary Info

### What Airship dependencies are you using?
  airship_flutter: 5.1.0

### What are the versions of any relevant development tools you are using?
Xcode 13.2
Flutter 2.8.0

# Report

### What unexpected behavior are you seeing?
The onShowInbox() and onShowInboxMessage() callbacks are not being called for iOS. They do work as expected for Android.

```
Airship.onShowInbox
    .listen((event) => debugPrint('Navigate to app's inbox'));

Airship.onShowInboxMessage
    .listen((messageId) => debugPrint('Navigate to message $messageId'));
```

### What is the expected behavior?
When when the user taps on a push notification that is linked to a message center message the  
onShowInboxMessage should get called.

### What are the steps to reproduce the unexpected behavior?
- run the app on iOS
- the app registers a Airship.onShowInboxMessage listener
- send a push notification combined with a message center message
- the iOS app gets the push
- tap on the push notification

Excepted: the onShowInboxMessage stream emits an event

Actual: The native message center and message view UI is displayed. The call is not called.

### Additional info

I noticed that in SwiftAirshipPlugin

```
    public func showMessage(forID messageID: String) {
        let event = AirshipShowInboxMessageEvent(messageID)
        AirshipEventManager.shared.notify(event)
    }
```
Is never called.

### Workaround
As a workaround, I am currently setting the MessageCenter displayDelegate manually on app startup and firing the events as follows:

```
import UIKit
import Flutter
import GoogleMaps
import airship_flutter
import AirshipKit

@UIApplicationMain
@objc class AppDelegate: FlutterAppDelegate {
    private let inboxCoordinator = InboxDelegate()
    
    override func application(
        _ application: UIApplication,
        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
    ) -> Bool {
        GeneratedPluginRegistrant.register(with: self)
        
        // This is a workaround to fix the fact that Airship Flutter is never calling the showMessage() and showInbox() callbacks
        DispatchQueue.main.async {
            MessageCenter.shared.displayDelegate = self.inboxCoordinator
        }
        
        return super.application(application, didFinishLaunchingWithOptions: launchOptions)
    }
}

class InboxDelegate:NSObject, MessageCenterDisplayDelegate {
    func displayMessageCenter(forMessageID messageID: String, animated: Bool) {
        print("displayMessageCenter \(messageID)")
        SwiftAirshipPlugin().showMessage(forID: messageID)
    }
    
    func displayMessageCenter(animated: Bool) {
        print("displayMessageCenter")
        SwiftAirshipPlugin().showInbox()
    }
    
    func dismissMessageCenter(animated: Bool) {
        print("displayMessageCenter")
    }
}

```

**Comment by oristanovic** [comment]
Thanks for the report, we will look into it

**Comment by khmMouna** [comment]
A PR was created, should be fixed with the next release. Thank you for the report

**Comment by rlepinski** [comment]
This was fixed in 5.1.1
